config/settings.yml

```diff
@@ -1839,3 +1839,14 @@ schema_contract:
 
 form_mock_ae_design_patterns:
   prefill: true
+
+vaos:
+  eps:
+    key_path: /fake/client/key/path
+    mock: false
+    client_id: "fake_client_id"
+    kid: "fake_kid"
+    audience_claim_url: "https://login.wellhive.com/oauth2/default/v1/token"
+    access_token_url: "https://login.wellhive.com/oauth2/default/v1/token"
+    api_url: "https://api.wellhive.com/care-navigation/v1"
+    scopes: "care-nav"
```

Comment:
Do you plan to add these settings to the manifest repo? Instructions [here](https://depo-platform-documentation.scrollhelp.site/developer-docs/vets-api-on-eks#VetsAPIonEKS-VetsAPIsettingsandsecrets) if you need them.

Reply:
yes, we have a separate ticket to add all configs and keys to manifest repo

Reply:
ok, sounds good. This PR looks good from my perspective, so @ me when you get a teammate review and I'll review.